.github/workflows/ci.yml

```diff
@@ -45,8 +45,14 @@ jobs:
           sudo cp unicode.mapping /etc/apache2/
           sudo mkdir -p /var/cache/modsecurity
           sudo chown -R www-data:www-data /var/cache/modsecurity
+      - name: first check config (to get syntax errors)
+        run: sudo apachectl configtest
       - name: start apache with module
-        run: |
-          sudo systemctl restart apache2.service
-          sudo cat /var/log/apache2/error.log
-
+        run: sudo systemctl restart apache2.service
+      - name: Show httpd error log
+        if: always()
```

Comment:
Unnecessary.
```suggestion
```

Reply:
I'm far from a specialist with this CI, but it seems to me that the error log won't be displayed in case of error if we remove that line. In case the service doesn't start, we want to know why, so it need to be displaye _always_.

Reply:
You're absolutely right.

Reply:
Is there any unsolved issue with this? If not, @theseion could you resolve this?

Reply:
As the review owner I cannot resolve comments. There's an open issue about this. It has to be either you or @marcstern, or you (temporarily) disable the merge check that requires all conversations to be resolved before merging.